Lib/Protocols/IdFTPServer.pas

```diff
@@ -6059,6 +6172,34 @@ procedure TIdFTPServer.DoOnDataPortBeforeBind(ASender: TIdFTPServerContext);
   end;
 end;
 
+function TIdFTPServer.GetPathSeperator : String;
+begin
+  Result := '/';
+  case FPathProcessing of

```

Comment:
What about `ftppCustom`? Do we need a way for the user to indicate a custom separator?

Reply:
We do in that case and I made two properties for the FTP Server for this.